lib/plugins/aws/package/compile/events/kafka.js

```diff
@@ -47,6 +47,16 @@ class AwsCompileKafkaEvents {
               minItems: 1,
               items: { $ref: '#/definitions/awsSecretsManagerArnString' },
             },
+            clientCertificateTLSAuth: {
```

Comment:
I went with this name over `clientCertificateTlsAuth`; thoughts?

Reply:
that sounds good, I see it maps nicely to the name used in CloudFormation :+1:

Reply:
One thing though - Should we use `Tls` or `TLS`? For CA Certificate I see you went with `Ca` instead of `CA` - it would be great to introduce consistency here - I think having `Tls` will be more in line with how `saslScram` is formatted

Reply:
yes that was what i was raising attention to. happy to got with `Tls`